src/styles/blocks/card.scss

```
@@ -0,0 +1,79 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  padding: 32px 16px 16px;
+  width: $card-width;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__img {
+    margin: 0 auto 40px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__item {
+    margin-bottom: 4px;
+    line-height: 18px;
+    font-size: 12px;
+    font-weight: 500;
+    color: $darker-text-color;
+  }
+
+  &__code {
+    margin-bottom: 16px;
+    line-height: 14px;
+    font-size: 10px;
+    color: $ligther-text-color;
+  }
+
+  &__review {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 24px;
+  }
+
+  &__review-number {
+    line-height: 14px;
+    font-size: 10px;
+    color: $darker-text-color;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 16px;
+  }
+
+  &__price-text {
+    font-size: 12px;
+    line-height: 18px;
+    color: $ligther-text-color;
+  }
+
+  &__price-value {
+    line-height: 18px;
+    font-size: 16px;
+    font-weight: 700;
+    color: $darker-text-color;
+  }
```

one more nesting level could be used
```suggestion
  &__price {
    display: flex;
    justify-content: space-between;
    margin-bottom: 16px;

    &-text {
      font-size: 12px;
      line-height: 18px;
      color: $ligther-text-color;
    }

    &-value {
      line-height: 18px;
      font-size: 16px;
      font-weight: 700;
      color: $darker-text-color;
    }
  }
```